ui-options.html

```diff
@@ -0,0 +1,158 @@
+                    <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the desktop view.
+                A separate panel bar for desktop displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-wideScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+
+        <div class="floe">
+            <div class="row">
+                <div class="columns small-12 medium-12 large-7">
+                    <header class="floe-header">
+                        <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
+                            <span class="floe-header-tagline">
+                                flexible learning for open education
+                            </span></a>
+                        </h1>
+                    </header>
+                </div>
+                <div class="columns small-12 medium-12 large-5 align-self-bottom">
+                    <nav class="floe-nav">
+                        <ul>
+                            <li><a href="./news/index.html">News</a></li>
+                            <li><a href="./resources.html">Resources</a></li>
+                            <li><a href="./ui-options.html">Learner UI Options</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+
+            <div id="content" class="floe-content">
+                <div class="flc-toc-tocContainer toc"> </div>
+
+                <!-- Begin UI Options Information Container -->
+                <article>
+                  <h2>About Learner / UI Options</h2>
+                  <p>
+                  Learner Options (or "UI Options") provides a way to enhance or improve website usability, flexibility, and accessibility by providing a way to customize and control aspects of a website without the need for additional software or tools. For those who design, build, or maintain websites, UI Options can also help reveal areas where the web content and structure can be changed to improve flexibility and robustness.
```

Comment:
In the last sentence you refer to it as "UI Options" maybe change that back to "Learner Options"
```suggestion
                  Learner Options (or "UI Options") provides a way to enhance or improve website usability, flexibility, and accessibility by providing a way to customize and control aspects of a website without the need for additional software or tools. For those who design, build, or maintain websites, Learner Options can also help reveal areas where the web content and structure can be changed to improve flexibility and robustness.
```